### ISSUE TITLE
Support emulation of plotter (like Sony PRN-C41)

### ISSUE THREAD

**Issue by MBilderbeek** [openMSX/openMSX#1579]
There are several (very similar) MSX plotters out there and there has been made some software for them as well. E.g. by Sony but also a few programs released by "Lunatic" (like Plotterdisk I and II). 

Would be fun to emulate such plotters and let the output be an SVG file :)

The command codes are available online in the Service Manual: https://archive.org/details/sonyprnc41sm/

**Comment by sndpl** [comment]
@MBilderbeek I think this issue can be closed?

**Comment by MBilderbeek** [comment]
Yes, this has been implemented via #2062 - THANKS AGAIN! Next time, please include a Closes #1579 thing, so it is cleaned up automatically :)
